src/makeCalculator.js

```diff
@@ -37,7 +37,29 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+
+    operate: function(action, number) {
```

Comment:
checkist - if you creating a method in the object, you don't need to use function keyword, use shortcut instead.